.github/CONTRIBUTING.md

```diff
@@ -41,6 +41,32 @@ Please read the rest of this document to ensure a smooth contribution process.
 
 Please see the [Contributor Guide in `MicrosoftDocs/PowerShell-Docs`](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md).
 
+#### Quick steps if you're changing an existing cmdlet
+
+If you made a change to an existing cmdlet and would like to update the documentation using PlatyPS,
+here are the quick steps:
+
+1. Install 
+`PlatyPS`
```

Comment:
Make sure you use markdownlint and correct all issue

Reply:
done.